app/dts/bindings/animations/zmk,animation-ripple.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2020, The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: |
+  Animated 'ripples' trigered by key presses.
+
+compatible: "zmk,animation-ripple"
+
+include: animation_base.yaml
+
+properties:
+  duration:
+    type: int
+    required: false
+    default: 1000
+    description: |
+      Approximate ripple travel time in milliseconds.
+
+  color:
+    type: int
+    required: true
+    description: |
+      Ripple color.
+
+  buffer-size:
+    type: int
+    default: 10
+    description: |
+      This will limit how many keystroke events the animation is able to track
```

Comment:
Should this be a KConfig? It does not seem like we would need to have different buffer sizes between each ripple driver instance

Reply:
Hmm, generally you're right, but then it's also tied to the `duration` prop - a longer animation will require a larger buffer.  
All in all, this seemed convenient and I'm not sure somebody would actually ever need to run two of these together... I think I'd leave it for now at least.